src/i18n/data/languages.json

```diff
@@ -0,0 +1,937 @@
+[
+  {
+    "code": "aa",
+    "name": "Afar",
+    "native": "Afar"
+  },
+  {
+    "code": "ab",
+    "name": "Abkhazian",
+    "native": "\u0410\u04a7\u0441\u0443\u0430"
```

Comment:
why the special characters ? not possible to avoid them ?

Reply:
I updated the file (formatting issue on Windows)